pom.xml

```diff
@@ -47,9 +70,9 @@
 						<id>spigot-repo</id>
 						<url>https://hub.spigotmc.org/nexus/content/groups/public/</url>
 				</repository>
-								<repository>
-						<id>devoted-repo</id>
+                                <!--<repository>
+						<id>teal-repo</id>
```

Comment:
Why do you want to remove teals repo so badly? Does it mess with dependencies? Dont we need that for AltManager and exilepearl?

Reply:
I have both of those cloned locally. This was going to be uncommented again in the next commit, just commented here because the build server was temporarily down.